eppo_core/src/lib.rs

```diff
@@ -80,5 +80,6 @@ pub use attributes::{
 pub use configuration::Configuration;
 pub use error::{Error, EvaluationError, Result};
 #[cfg(feature = "event_ingestion")]
+pub use event_ingestion::ContextValue;
```

Comment:
Is there any reason you want to expose it at the top level instead of keeping it scoped to `event_ingestion` mod?